skrobot/coordinates/base.py

```diff
@@ -1018,6 +1023,14 @@ def assoc(self, child, relative_coords=None,
                 'Please use `relative_coords` instead',
                 DeprecationWarning)
             relative_coords = kwargs['c']
+
+        isInvalidAssoc = (child.parent is not None) and (not force)
```

Comment:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names
We should use lower case and underscores for variable names.

Reply:
ok, I fixed it!